Prepare directory structure for V1

decentralgabe opened the pull request.

dtmcg commented:
do you want to keep a working copy under spec/spec.md?

bumblefudge commented:
Some thoughts, each of which I'd recommend converting to new issues for addressing post v1 unless they have easy answers or can be addressed by quick edits:
1. why `author` and not, say, `controller`?
2. I took a kind of contrarian, sassy stab at the "todo" to define trust topics, please iterate if it's too sassy or judgmental. Spec is a little cryptic without a little more explanation of its use-case and mental model!
3. The Topic Registry TODO I think should be done before v1.  Since topics are defined as URIs of schemas, you could just point to [schema-directory](https://github.com/decentralized-identity/schema-directory/blob/main/content/schemas.md) or something equivalent at, say, TOIP?  
3B. Sidenote: the schema directory doesn't really look like the [other DIF registries](https://identity.foundation/claim-format-registry/#registry), maybe we can give that group a nudge to make something more... presentable? It feels like another case of the "v1 can't be cut if it depends on a v0 spec or registry" problem that has plagued this WG since time immemorial 😉

bumblefudge commented:
also note
![image](https://user-images.githubusercontent.com/37127325/205076155-4aeb7c45-af6a-42d1-a1f9-22aad39a3e80.png)

bumblefudge commented on spec/v1/spec.md:
```diff
@@ -503,25 +501,33 @@ This document specifies neither _entity_ nor _topic_ at the document level, and
 
 ::: todo Topic Registry
   Create a place for topics to be registered.
+  The [schema directory](https://github.com/decentralized-identity/schema-directory/blob/main/content/schemas.md) seems like one place where schemas are registered, but I would also recommend adding a definition of what you mean by "registry" and ideally more than one example. __chair
```
This should probably be addressed before v1; see PR discussion for more detail

decentralgabe commented:
Thanks @bumblefudge for the review.

I opened up https://github.com/decentralized-identity/schema-directory/issues/5 on the Schema Directory repo.

I've added spec text around Trust Topics that I hope addresses your feedback.

I wonder about the semantic different between _author_ and _controller_. I believe _author_ implies _controller_ -- or at least that is my intention, though I'm not sure the inverse is true. The intent was to establish who created the document.

Got rid of those TODOs. Let me know what you think.

decentralgabe commented on spec/v1/spec.md:
```diff
@@ -527,8 +522,14 @@ There may be different motivations for interacting with [[ref:Trust Establishmen
 
-::: todo Topic Registry
-  Create a place for topics to be registered.
+As per the section on [Publication & Discovery](#publication-discovery) there are a number of possibilities for hosting [[ref:Trust Establishment Documents]]. A related concept is defining a set of well-known [[ref:Topics]] that can find utility across an ecosystem of trust. One such option in housing these well-defined [[ref:Topics]] is a _Topic Registry_. Topic Registries can be as simple as [G ...
```
@csuwildcat let me know what you think about this section, specifically on the accuracy of the DWN text

decentralgabe commented on spec/v1/spec.md:
```diff
@@ -527,8 +522,14 @@ There may be different motivations for interacting with [[ref:Trust Establishmen
+
+Another project at DIF, [Decentralized Web Nodes](https://identity.foundation/decentralized-web-node/spec/), also provide the ability to host and address multiple types of data. Taking advantage of the [Collections interface](https://identity.foundation/decentralized-web-node/spec/#collections) it is possible to create a _Trust Topic Collection_ that is exposed via [Feature Detection](https://ide ...
+
+::: todo
```
@TelegramSam could use your magic here
